rules-reviewed/technology-usage/database-technology-usage.windup.xml

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0"?>
+<ruleset id="technology-usage-database" xmlns="http://windup.jboss.org/schema/jboss-ruleset" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://windup.jboss.org/schema/jboss-ruleset http://windup.jboss.org/schema/jboss-ruleset/windup-jboss-ruleset.xsd">
+    <metadata>
+        <description>
+            This ruleset provides statistical summaries of the database items that were found during the analysis.
+        </description>
+        <dependencies>
+            <addon id="org.jboss.windup.rules,windup-rules-javaee,2.7.0.Final" />
+            <addon id="org.jboss.windup.rules,windup-rules-java,2.7.0.Final" />
+        </dependencies>
+        <phase>PostMigrationRulesPhase</phase>
+    </metadata>
+    <rules>
+        <rule id="technology-usage-database-01000">
+            <when>
+                <graph-query discriminator="JdbcDatasourceModel"  />
+            </when>
+            <perform>
+                <technology-identified name="JDBC datasources">
+                    <tag name="Store"/>
+                    <tag name="Database"/>
+                    <tag name="Java EE"/>
+                </technology-identified>
+            </perform>
+        </rule>
+        <rule id="technology-usage-database-01001">
+            <when>
+                <graph-query discriminator="DataSourceModel">
+                    <property name="isXA">true</property>
```

Comment:
This rule is failing its test because this condition is not working properly unless you specify that the property is not a string but a boolean.
You can do it adding the `type` attribute in this way:
```xml
                    <property name="isXA" type="BOOLEAN">true</property>
```
This was an issue raised during #270

Reply:
Thanks Marco, I didn't realize that